.github/workflows/deploy-preview.yml

```diff
@@ -21,46 +19,40 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           version: 8
-      - name: Use Node.js 18
-        uses: actions/setup-node@v4
+      - uses: actions/setup-node@v4
```

Comment:
Similar to code comments, I only like them when they are stating a reason that's not obvious. Otherwise they're superfluous.

Reply:
Makes sense